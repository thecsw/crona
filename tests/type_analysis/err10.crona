foo : byte() {
    a : int;
    return a;
}

bar : bool() {
    return 10;
}

rawr : int() {
    a : byte;
    return a; // no error
}

ss : string() {
    return 90;
}

sss : string() {
    return true;
}

oof : bool() {
    return "string";
}

main : void() {}