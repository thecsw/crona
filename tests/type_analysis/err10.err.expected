FATAL [3,12]: Bad return value
FATAL [7,12]: Bad return value
FATAL [16,12]: Bad return value
FATAL [20,12]: Bad return value
FATAL [24,12]: Bad return value
Type Analysis Failed
